test/dataset/test_pandas.py

```diff
@@ -228,16 +228,3 @@ def test_long_csv_3M():
         assert entry["start"] == expected_entry["start"]
         assert entry["item_id"] == expected_entry["item_id"]
         assert np.allclose(entry["target"], expected_entry["target"])
-
-    with io.StringIO(data) as fp:
-        ds = pandas.PandasDataset.from_long_dataframe(
-            pd.read_csv(fp, index_col="timestamp"),
```

Comment:
I removed this part of the test since I don't think it's realistic to load a long dataframe this way, since timestamps can be repeated: it only makes sense to set this column as index after the `groupby` in general.

Reply:
An index in pandas does not need to be unique.